packages/plugins/gatsby-preview/admin/src/pages/SettingsPage/utils/api.js

```diff
@@ -0,0 +1,29 @@
+import axiosInstance from '../../../utils/axiosInstance';
+import pluginId from '../../../pluginId';
+
+export const fetchContentTypes = async () => {
+  try {
+    const {
+      data: { data },
+    } = await axiosInstance.get('/content-manager/content-types');
```

Comment:
Is it recommend for plugin developers to use these internal endpoints? I'm asking because it's an official plugin developers might look at it and copy it's behavior, which will make it very hard to us, to change these internal APIs in the future.

Reply:
Well at some point all models should be available in the entire app, here since we are hooking into the CM it is normal to use the CM's content types api.

Reply:
I agree with you. I guess we just have to be aware of that fact, if we advertise it. Another, more long-term solution, could be to expose the content types through the public API ...